### PR TITLE
adding shaktiman file

### PR DESCRIPTION
**Pull request by shaktimancodes**
please review

### CODE REVIEW

File: shaktimanfile.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+everyone knows that
+rohit is great
```
Comment (rohitkr7) [inline shaktimanfile.md]: file looks good but replace rohit with shaktiman

**Review by rohitkr7 — CHANGES_REQUESTED** [review]

File: shaktimanfile.md (hunk 1 above)
Comment (shaktimancodes) [reply shaktimanfile.md]: modified the file as suggested

please check now

Comment (rohitkr7) [reply shaktimanfile.md]: cool thanks

**Review by rohitkr7 — APPROVED** [review]
go ahead and merge it I am approving it.

**Pull request merged**